pageserver/src/tenant/timeline.rs

```diff
@@ -2811,6 +2505,8 @@ impl Timeline {
                 // 3. rename to the final name
                 // 4. fsync the parent directory.
                 // Note that (1),(2),(3) today happen inside write_to_disk().
+                //
+                // FIXME: writer has already fsynced contents, we only need to fsync the rename
                 par_fsync::par_fsync(&[new_delta_path]).context("fsync of delta layer")?;
```

Comment:
pre-existing issue, right? Please remove comment and turn into a follow-up issue.
Also I commented somewhere else on fsync dir contents. Same thing.

Reply:
I think these are good to leave, otherwise we will just forget them once more.